product_attribute_company_favorite/models/product_attribute.py

```diff
@@ -24,26 +18,39 @@ def _name_search(
 
     @api.model_create_multi
     def create(self, vals_list):
-        new_attributes_favorite_for_all_companies = (
+        new_record_favorite_for_all_companies = (
             self.env["ir.config_parameter"]
             .sudo()
-            .get_param(
-                "product_attribute_company_favorite.new_attributes_favorite_for_all_companies"
-            )
+            .get_param(self.CREATION_ACROSS_COMPANY_CONFIG)
         )
-        attributes = super().create(vals_list)
-        if new_attributes_favorite_for_all_companies:
-            for attribute in attributes:
+        records = super().create(vals_list)
+        if new_record_favorite_for_all_companies:
+            for record in records:
                 company_ids = (
                     self.env["res.company"]
                     .with_context(active_test=False)
                     .search([])
                     .ids
                 )
                 for company_id in company_ids:
-                    contextual_attribute = attribute.with_company(company_id)
-                    contextual_attribute.is_favorite = True
-        return attributes
+                    contextual_record = record.with_company(company_id)
+                    contextual_record.is_favorite = True
+        return records
+
+
+class ProductAttribute(models.Model):
+    _inherit = ["product.attribute", "product.attribute.mixin"]
+    _name = "product.attribute"
+
+    CREATION_ACROSS_COMPANY_CONFIG = "product_attribute_company_favorite.new_attribute_favorite_for_all_companies"  # noqa
```

Comment:
You may split string I think to avoid #noqa


```suggestion
    CREATION_ACROSS_COMPANY_CONFIG = "product_attribute_company_favorite."
    "new_attribute_favorite_for_all_companies" 
```

Idem in other strings in the PR.

You may also have shorter word like CREATION_ACROSS_CPNY (not mandatory)

Reply:
Fixed !